unstructured/file_utils/filetype.py

```diff
@@ -342,9 +342,19 @@ def _detect_filetype_from_octet_stream(file: IO) -> FileType:
         elif all(f in archive_filenames for f in EXPECTED_PPTX_FILES):
             return FileType.PPTX
 
+    if LIBMAGIC_AVAILABLE:
+        return magic.from_buffer(file.read(4096), mime=True)
```

Comment:
Could we add handling so that if this is `None`, we still return `FileType.UNK`?